content/docs/about/use-cases.md

```diff
@@ -0,0 +1,30 @@
++++
+title = "Use Cases"
+description = "Kubeflow use cases"
+weight = 1
+aliases = ["/docs/", "/docs//", "/docs/usecases/"]
++++
+
+## General Overview
+
+The end goal of every organization is to have their ML model successfully running in production  and generating value to the business. But what does it take to reach that point? Before a model ends up in production, there are potentially many steps required to build and deploy a ML model: data loading, verification, splitting, processing, feature engineering, model training and verification, hyperparameter tuning, and model serving. In addition, ML models can require more observation than traditional applications, because your data inputs can drift over time. Manually rebuilding models and data sets is time consuming and error prone. To simplify these requirements and challenges, we introduce Kubeflow.
+
+## Deploying and managing a complex ML system at scale
+
+Kubeflow is a scalable, portable, distributed ML platform that runs on Kubernetes, meaning that all capabilities of Kubernetes are available to a Kubeflow user. With Kubeflow you can manage the entire AI organization at scale and still be able to maintain the same quality of control. Kubeflow and Kubernetes provide consistent and efficient operations and optimized infrastructure utilization. This means that your top researchers have more time to focus on the valuable tasks of developing domain specific intellectual property rather than debugging DevOps configuration issues. Kubeflow’s Core and EcoSystem Critical User Journeys (CUJs) provide software solutions for end-to-end workflows i.e. build, train and deploy and/or develop a model and create, run and explore a pipeline.
+
+## Experimentation with training an ML model
+
+Rapid experimentation is critical to building high quality machine learning models quickly. Kubeflow offers a user-friendly UI interface that allows you to track and compare experiments. You can decide later on which experiment was the best and use it as a main source for your future steps. On top of that Kubeflow 1.0 provides stable software sub-systems for model training including Jupyter notebooks, Fairing, popular ML training operators such as Tensorflow and Pytorch that run efficiently and securely in Kubernetes isolated namespaces. The ML training operators simplify configuration and operations of scaling ML training tasks. In addition, the Kubeflow has delivered Critical User Journeys(CUJs), such as the build, train and deploy, which provide end-to-end workflows that speed development. Please let more on the CUJs [here](http://bit.ly/kf_roadmap).
```

Comment:
Change this:

```
Please let more on the CUJs [here](http://bit.ly/kf_roadmap).
```

To this:

```
You can read more about the CUJs in the [Kubeflow roadmap](http://bit.ly/kf_roadmap).
```

Reasoning: See Joe's talk about addressing the reader in tech docs. Also, it's better in link text to let the reader know where the link will take them, rather than using "here".